metadata.tvmaze/libs/data_utils.py

```diff
@@ -165,8 +165,16 @@ def add_main_show_info(list_item, show_info):
         'episodeguide': str(show_info['id']),
     }
     if show_info['network'] is not None:
-        video['studio'] = show_info['network']['name']
-        video['country'] = show_info['network']['country']['name']
+        country = show_info['network']['country']
+        video['studio'] = show_info['network']['name'] + ' (' + country['code'] + ')'
```

Comment:
Please use string .format() method instead of concatenating. Also please use unicode literals for user-facing strings that may contain non-ASCII characters.

Reply:
Changed